### PR TITLE
Pull request/bfa2ac61

### PR DESCRIPTION
**Pull request by thofma**

### CODE REVIEW

File: Hecke/versions/0.1.3/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.4
+Nemo 0.4.1
```
Comment (tkelman) [inline Hecke/versions/0.1.3/requires]: does this need an upper bound of 0.5?

**Comment by tkelman** [comment]
If you're including GPL'ed C code in the repository, you should point that out in your top-level license file.

File: Hecke/versions/0.1.3/requires (hunk 1 above)
Comment (thofma) [inline Hecke/versions/0.1.3/requires]: No, I don't think so.

**Comment by thofma** [comment]
I changed the license.

**Comment by tkelman** [comment]
You're not the original author or copyright holder of that file are you? It says Fredrik Johansson is. IANAL, but unless you have the author's permission to relicense the entire file, I don't think the change in https://github.com/thofma/Hecke.jl/commit/e88d103573675b87a072355a250bf6b6d00b4d02 is legal.

**Comment by thofma** [comment]
He actually never was the copyright holder. Someone copy pasted the wrong header in the wrong file. 

Anyway, I talked to the people involved yesterday (in person).

File: Hecke/versions/0.1.4/sha1 [hunk 2]
```diff
@@ -1 +1 @@
-47b6e26ee4dd2c319ca19c8e12162ff2616bcb1c
+e88d103573675b87a072355a250bf6b6d00b4d02
```
Comment (tkelman) [inline Hecke/versions/0.1.4/sha1]: the metadata tag should be consistent with the git tag at https://github.com/thofma/Hecke.jl/releases

Comment (thofma) [inline Hecke/versions/0.1.4/sha1]: Should be consistent now.

**Pull request merged**